params.go

```diff
@@ -215,6 +217,12 @@ func (p *Params) AddExpand(f string) {
 	p.Expand = append(p.Expand, &f)
 }
 
+// SetUsage sets the usage field on the Params struct.
+// Unstable: for internal stripe-go usage only.
+func (p *Params) SetUsage(usage []string) {
```

Comment:
Consider making this function receiverless and renaming it to something more scary (InternalSetUsage?).

Reply:
There are no usages of this method.